external/docker-integration-tests/pom.xml

```diff
@@ -34,6 +34,13 @@
     <sbt.project.name>docker-integration-tests</sbt.project.name>
   </properties>
 
+  <repositories>
+    <repository>
+      <id>db2</id>
+      <url>https://app.camunda.com/nexus/content/repositories/public/</url>
```

Comment:
Whoops, sorry, had the wrong path: https://app.camunda.com/nexus/content/repositories/public/com/ibm/db2/jcc/

This seems fine but I'll be really mad if this repository ever goes down and breaks the build :smile: